src/components/Strategies/StrategyDetails.tsx

```diff
@@ -19,6 +19,7 @@ import { strategies } from './strategyData';
 import { blockConfig, Repeat } from './Blocks';
 
 const { Title, Text } = Typography;
+const web3 = new Web3(ethereum as any);
```

Comment:
maybe we can put this line little above? on top of line 19 
(import { blockConfig, Repeat } from './Blocks';) 
for example?

Reply:
You mean above imports? Nothing should come above imports. In native ES6 that would not even compile. Why do you think it should go there?